samples/word/99-preview-apis/manage-fields.yaml

```
@@ -0,0 +1,151 @@
+id: word-manage-fields
+name: Get fields
+description: This sample shows how to get existing fields.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#get-first").click(() => tryCatch(getFirstField));
+        $("#get-parent-body").click(() => tryCatch(getParentBodyOfFirstField));
+        $("#get-all").click(() => tryCatch(getAllFields));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function getFirstField() {
+          await Word.run(async (context) => {
+            let field = context.document.body.fields.getFirstOrNullObject();
```

Here and elsewhere, consider making this `const` since it doesn't get reassigned.
```suggestion
            const field = context.document.body.fields.getFirstOrNullObject();
```